x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/inference/utils/StatisticsTests.java

```diff
@@ -30,4 +31,20 @@ public void testSoftMaxWithNoValidValues() {
         expectThrows(IllegalArgumentException.class, () -> Statistics.softMax(values));
     }
 
+    public void testSigmoid() {
+        double eps = 0.000001;
+        List<Tuple<Double, Double>> expectations = Arrays.asList(
```

Comment:
Maybe rename to `argumentAndExpectedResult` or `inputAndExpectedOutput`? I know it's lengthy but `expectations` does not really tell what this list is about without looking at the assertion below.

Alternatively, you could get rid of this list and rewrite assertions as:
```
assertThat(Statistics.sigmoid(0.0), closeTo(0.5, eps));
```

It's a little bit code duplication but I thinks acceptable in tests.